cmd/quadlet/main.go

```diff
@@ -150,12 +153,17 @@ func loadUnitsFromDir(sourcePath string, units map[string]*parser.UnitFile) {
 			Debugf("Loading source unit file %s", path)
 
 			if f, err := parser.ParseUnitFile(path); err != nil {
-				Logf("Error loading '%s', ignoring: %s", path, err)
+				if prevError != nil {
+					Logf("%s", prevError)
+				}
+				prevError = fmt.Errorf("Error loading '%s', %w", path, err)
```

Comment:
```suggestion
				err = fmt.Errorf("Error loading '%s', %w", path, err)
				if prevError != nil {
					prevError = fmt.Errorf("%s\n%s", prevError, err)
				} else {
				        prevError = err
				}
```

The proposed logic would seem a bit saner to me.  This way, all errors are carried over in `prevError` and get logged in `main`.

I think we need to be careful with logging errors now to avoid duplicates.